public/javascripts/notebook/sidebar.js

```diff
@@ -31,16 +31,24 @@ require(["jquery", "jquery.gridster"], function($, gridster) {
 
   site.scroll(sticky_relocate);
 
-  ul.gridster({
-    max_cols: nbCols,
+  var g = ul.gridster({
     widget_margins: [m, m],
-    widget_base_dimensions: [b, 250],
-    max_sizex: 2,
+    widget_base_dimensions: [b, 50],
+    max_size_x: 2,
     helper: 'clone',
     resize: {
       enabled: true
     }
-  });
+  }).data("gridster");
+
+  var sizeContent = function(id) {
+    var td = $("#"+id);
+    var wid = td.parents(".widget:first");
+    td.height((wid.data().sizey * 50 /*compute widget height*/) - (28+2*4 /*h4*/) + "px");
+  };
```

Comment:
this number magic stinks ;)
wouldn't be surprised if such&related sidebar relocation could be slowing down the browser...